loggly/loggly.go

```diff
@@ -63,11 +63,14 @@ type Adapter struct {
 	client http.Client
 	tags   string
 	log    *log.Logger
+	msgs   []logglyMessage
+	url    string
+	m      sync.Mutex
```

Comment:
I believe the appropriate style is to have the mutex defined before the data structure that it guards.

e.g.

m       sync.Mutex
msgs []logglyMessage